main.js

```diff
@@ -1,6 +1,8 @@
+'use strict';
+
 // Dependencies
 const { token } = require('./config.json');
```

Comment:
Usually `.json` is omitted, but quite a number of people prefer to leave it there for clarity.